OCR/frontend/tsconfig.json

```diff
@@ -3,5 +3,8 @@
   "references": [
     { "path": "./tsconfig.app.json" },
     { "path": "./tsconfig.node.json" }
-  ]
+  ],
+  "compilerOptions": {
+    "lib": ["ESNext.Array"]
+  }
```

Comment:
whats the purpose of this? I not sure what it does exactly

Reply:
It cleared out a TS error pertaining to using `Array.prototype.toSorted` ([mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)) which I'm using in `SortedTable` because the `sort` method is an in place sort and we don't want to mutate the provided data.